ops/docker/ci-builder/Dockerfile

```
@@ -58,13 +58,15 @@ RUN go install github.com/ethereum/go-ethereum/cmd/geth@$(jq -r .geth < versions
 RUN go install gotest.tools/gotestsum@latest
```

The latest gotestsum is https://pkg.go.dev/gotest.tools/gotestsum@v1.11.0
```suggestion
RUN go install gotest.tools/gotestsum@v1.11.0
```